manuscript.md

```diff
@@ -406,9 +406,17 @@ be no interaction between two species, it will be correct in the majority of
 cases because the majority of potential interactions in a network typically
 do not exist.  Therefore this "empty-matrix" model would always have an
 _accuracy_ of $1-C$, where $C$ is the observed connectance, which would almost
-always be greater than 50%. In response to the general problem of biases in
+always be greater than 50%. Understanding model performance within 
+sensitivity-specificity space may be more informative, where sensitivity 
+evaluates how good the model is at predicting true interactions (TPs) and 
+specificity refers to the prediction of true non-interactions (TNs).
```

Comment:
Does it make sense to move the caveat regarding true negatives that is discussed in the table caption here?

Reply:
yes

Reply:
also i think we need to change specificity/sensitivity to their rate synonyms, or at least add a parenthetical